src/components/Class/index.js

```diff
@@ -44,7 +44,7 @@ function Class() {
   const { user : { rolesList = [] } } = useSelector(({User}) => User.data)
```

Comment:
I think we can remove the logic from line number 44 to 49. We are now giving access to create a class to anyone. cc @pysaquib

Reply:
Done